{% extends "layout.html" %}

{% block page_content %}
<link rel="stylesheet" href="{{ url_for('static', filename='css/pages/common.css') }}">
<link rel="stylesheet" href="{{ url_for('static', filename='css/pages/wetlab.css') }}">

<div class="our-team-section">
  <div class="gradient"></div>
  <div class="overlay-text">
    <h3>LAB</h3>
    <h1>DESIGN</h1>
  </div>
</div>


<div class="stage">
  <div class="layout">

    <!-- 左侧导航 -->
    <aside class="sidebar">
      <div class="indicator" id="indicator"></div>
      <div class="sidebar-inner">
        <h3 class="side-title">OVERVIEW</h3>

        <!-- Content Start -->
        <nav>
          <ul class="menu" id="menu">
            <li><a class="navlink" href="#introduction">INTRODUCTION</a></li>
            <li><a class="navlink" href="#strategy1">STRATEGY 1</a></li>
            <li><a class="navlink" href="#strategy2">STRATEGY 2</a></li>
            <li><a class="navlink" href="#strategy3">STRATEGY 3</a></li>
            <li><a class="navlink" href="#pathway">PATHWAY <br>CONSTRUCTION</a></li>
            <li><a class="navlink" href="#references">REFERENCES</a></li>
          </ul>
        </nav>
      </div>
    </aside>

    <!-- 右侧正文 -->
    <main class="content">
      <div class="paper">

        <!-- ID：introduction -->
        <section id="introduction">
          <h1>INTRODUCTION</h1>
          <p class="sub">Edwardsiella piscicida is an intracellular parasitic pathogenic bacterium belonging to the
            genus Edwardsiella. It is a primary causative agent of edwardsiellosis in marine farmed fish worldwide,
            capable of causing large-scale mortality in economically important fish species such as turbot (Scophthalmus
            maximus). This bacterium inflicts significant economic losses on the aquaculture industry.</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design1.webp" alt="design1">
          <p class="figurecaption">Figure 1 Schematic diagram of Edwardsiella piscicida invading Scophthalmus maximus
          </p>

          <p class="sub">Fish vaccines share a similar mechanism with vaccines administered to humans. They involve
            introducing specially treated pathogenic microorganisms (such as bacteria, viruses, etc.) or their
            metabolites into fish bodies via methods like injection and immersion. This process stimulates the fish's
            own immune system to produce antibodies, thereby protecting the fish against infections caused by the
            corresponding diseases.</p>
          <p class="figurecaption">Table 1 comparison of three vaccination routes for fish vaccines</p>
          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/table1.webp" alt="table1">

          <p class="sub">In the immunization operations of fish vaccines, immersion immunization is the simplest
            immunization method. It can meet the efficiency requirements of large-scale farming, reduce the stress and
            mortality risks during the fry stage, and cover the immunization needs across the entire developmental
            stages of fish. However, the immersion method still has limitations, including relatively low immune
            protection rate and short duration of immunity.</p>

          <p class="sub">In the preliminary study, we subjected zebrafish to immersion infection using fluorescent
            protein-labeled Edwardsiella piscicida, and acquired imaging data via a fluorescence microscope. Image
            analysis results showed that distinct fluorescent signals were observable in the zebrafish body, confirming
            that the fluorescent protein-labeled Edwardsiella piscicida had successfully invaded the zebrafish host,
            colonized relevant tissues, and achieved effective dissemination. This provides direct visual evidence for
            the infectivity of the bacterium under immersion conditions.
          </p>
          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design2.webp" alt="design2">
          <p class="figurecaption">Figure 2 Fluorescence Microscopic Images of Zebrafish Infected with Fluorescent
            Protein-Labeled Edwardsiella piscicida</p>

          <p class="sub">Based on the aforementioned research, we plan to replace Edwardsiella piscicida with
            Escherichia coli to construct an engineered strain capable of linear movement under blue light and tumbling
            under green light. Furthermore, we will integrate a light irradiation device to achieve precise regulation
            of the engineered strain's movement states, thereby enhancing the invasive capacity of the vaccine strain.
            Additionally, we will verify the efficacy of this light-driven micro-nanorobot using equipment such as
            fluorescence microscopes.</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design3.webp" alt="design3">
          <p class="figurecaption">Figure 3 Schematic Diagram of Tumbling and Linear Motion of the Engineered Strain</p>
        </section>


        <!-- ID：strategy1 -->
        <section id="strategy1">
          <h1>Strategy 1:The Flagellar Movement Control System in Escherichia coli</h1>
          <p class="sub">The flagellum of Escherichia coli consists of several key components:
            Flagellar Motor: Converts the chemical energy from ion gradients (H⁺/Na⁺) into mechanical rotational force
            to drive flagellar rotation. It includes:
            MS Ring: The starting point of torque transmission, connecting the C Ring (direction switch) and the Rod.
          </p>
          <ul>
            <li>C Ring: Controls the rotation direction (clockwise/counterclockwise) and responds to chemical signals.
            </li>
            <li>Stator (e.g., MotA/B): A transmembrane ion channel that generates torque using proton flow to push the
              MS Ring to rotate.</li>
            <li>Rod: Transmits the motor's torque to the Hook and requires high rigidity to resist torsional
              deformation.</li>
            <li>LP Ring: Supports the rotation of the Rod and reduces friction loss.</li>
            <li>Hook: A flexible "universal joint" that connects the Rod and the Filament, transmitting torque and
              buffering deflection stress.</li>
            <li>Export Apparatus: Assembles flagellar components (similar to Type III secretion system, T3SS) and serves
              as the assembly base for the Rod.</li>
          </ul>

          <p class="sub">Like numerous other biological signaling systems, the E.coli chemotaxis pathway relies on
            reversible protein phosphorylation as its signaling mechanism.That said, the core signaling biochemistry
            differs somewhat between prokaryotes and eukaryotes.CheA functions as a kinase that utilizes ATP to undergo
            autophosphorylation at a particular histidine residue. Subsequently, phospho-CheA molecules act as donors in
            autokinase reactions, transferring phosphoryl groups to specific aspartate residues within CheY.</p>

          <p class="sub">Phosphorylated CheY promotes clockwise (CW) rotation of the flagellar motors, while
            phosphorylated CheB exhibits high methylesterase activity toward methyl-accepting chemotaxis proteins
            (MCPs). These activated response regulators have a brief lifespan due to their rapid loss of phosphoryl
            groups through spontaneous self-hydrolysis. Additionally, CheZ accelerates the dephosphorylation of
            phospho-CheY, ensuring that locomotor responses to fluctuations in the availability of signaling phosphoryl
            groups to CheY occur rapidly.</p>

          <p class="sub">In terms of the molecular mechanism underlying the direction switching of the flagellar motor,
            when there is no CheY protein binding, the stator units that rotate clockwise are located outside the FliGCC
            subring of the C ring, driving the flagellar motor to rotate counterclockwise. When CheY binds to the C
            ring, it promotes a conformational change in the C ring, which contracts toward the center of the flagellar
            motor and shifts upward as a whole. This further induces a conformational change in the FliGCC domain,
            ultimately leading to a 180° flip of the αtorque on FliGCC. Consequently, the stator units move from the
            outside to the inside of FliG, undergoing spatial relocation on the inner membrane, which prompts the C ring
            to rotate clockwise, thereby achieving the direction switching of the flagellar motor.</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design4.webp" alt="design4">
          <p class="figurecaption">Figure 4 Schematic Diagram of Flagellar Rotation Direction Regulated by Partial
            Bacterial Chemotaxis Networks</p>
        </section>

        <!-- ID：strategy2 -->
        <section id="strategy2">
          <h1>Strategy 2: Blue Light-Controlled Linear Movement of Escherichia coli</h1>
          <p class="sub">The pDawn system functions as a reliable gene expression platform regulated by light
            activation. Specifically, blue light triggers the expression of target genes, with this effect being
            significantly amplified by either higher light intensities or prolonged exposure durations. The underlying
            molecular mechanism of this system is explained as follows. </p>

          <p class="sub">A key component of the system is the histidine kinase YF1, which contains a
            light-oxygen-voltage (LOV) domain sensitive to blue light. In the dark (without blue light exposure), YF1
            undergoes autophosphorylation and transfers the phosphate group to its paired response regulator FixJ.
            Phosphorylated FixJ then activates substantial transcription from the FixK2 promoter, a process that is
            inhibited when YF1 absorbs blue light.</p>

          <p class="sub">All elements of the YF1/FixJ two-component system (TCS) are integrated into the medium-copy
            plasmid pDawn. Here, YF1 and FixJ are constitutively expressed as a bicistronic operon under the control of
            the LacIq promoter. Target genes can be inserted in a single cloning step through a multiple-cloning site
            (MCS) located downstream of the pFixK2 promoter. Additionally, the λ phage repressor cI, whose expression is
            driven by FixK2, inhibits the λ promoter pR within pDawn. This configuration reverses the signal polarity,
            resulting in light-dependent activation of gene expression: when blue light is present, FixK2 activity is
            suppressed, leading to reduced cI production. Consequently, the λ promoter pR is derepressed, initiating
            transcription of the target gene.</p>

          <p class="sub">Thus, we utilize the pDawn system for the expression of the CheZ gene. Under blue light
            induction, the CheZ protein is expressed, which induces ccw rotation of the flagella and promotes the linear
            movement of bacteria.</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design5.webp" alt="design5">
          <p class="figurecaption">Figure 5 Linear Motion Induced by Blue Light</p>
        </section>

        <!-- ID：strategy3 -->
        <section id="strategy3">
          <h1>Strategy 3: Green Light-Controlled Tumbling Motion of Escherichia coli</h1>
          <p class="sub">The second light sensor was engineered by cloning the Synechocystis PCC 6803 ccaS-ccaR-cpcG2
            genomic cluster into a ColE1-origin plasmid, and replacing the native cpcG2 output gene with CheA or
            CheY.CcaS is a SK with a N-terminal PCB-binding cyanobacteriochrome GAF domain and a C-terminal histidine
            kinase domain.CcaS holoprotein is produced in a green absorbing, phosphatase active ground state (Pg). Green
            light switches CcaS Pg to a kinase active Pr conformation, where it phosphorylates the CcaR.CcaR∼P binds to
            a G-box operator within PcpcG2,activating transcription.</p>

          <p class="sub">In this project, the CcaS/CcaR dual-plasmid green light system was used to express CheA and
            CheY proteins respectively, thereby achieving the goal of regulating the tumbling movement of engineered
            strains via green light.</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design6.webp" alt="design6">
          <p class="figurecaption">Figure 6 Tumbling Motion Induced by Green Light</p>

        </section>

        <!-- ID：pathway-->
        <section id="pathway">
          <h1>Pathway construction</h1>
          <h4>GROUP 1: pBBR1MCS-5-CheA、pBBR1MCS-5-CheY、pBBR1MCS-5-CheZ</h4>
          <p class="sub">
            Group 1 comprises three plasmids.The pBBR1MCS-5 vector was used to express the CheA, CheY, and CheZ genes
            respectively, which served to familiarize with the experimental procedure and conduct a preliminary
            exploration of bacterial motility.
          </p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design7.webp" alt="design7">
          <p class="figurecaption">Figure 7 schematic diagram of pBBR1-MCS-CheA</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design8.webp" alt="design8">
          <p class="figurecaption">Figure 8 schematic diagram of pBBR1-MCS-CheY</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design9.webp" alt="design9">
          <p class="figurecaption">Figure 9 schematic diagram of pBBR1-MCS-CheZ</p>

          <h4>GROUP 2: pBBR1MCS-5-CheA-Flag、pBBR1MCS-5-CheY-Flag、pBBR1MCS-5-CheZ-Flag</h4>
          <p class="sub">Group 2 comprises three plasmids.Building upon Group 1, Flag tags were respectively added to
            the three target genes, which were used in Western blot to verify the protein expression status.</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design10.webp" alt="design10">
          <p class="figurecaption">Figure 10 schematic diagram of pBBR1-MCS-CheA-Flag</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design11.webp" alt="design11">
          <p class="figurecaption">Figure 11 schematic diagram of pBBR1-MCS-CheY-Flag</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design12.webp" alt="design12">
          <p class="figurecaption">Figure 12 schematic diagram of pBBR1-MCS-CheZ-Flag</p>

          <h4>GROUP 3:pDawn-CheZ、pDawn-CheZ-Flag、pDawn-sfGFP</h4>
          <p class="sub">Group 3 comprises three plasmids, which express CheZ, CheZ-Flag, and sfGFP respectively under
            blue light induction. Among them, sfGFP is used for fluorescence detection with a multi-functional
            microplate reader to confirm the successful construction of the expression system and conduct
            semi-quantitative evaluation of expression intensity, thereby providing a reference for the optimization of
            induction conditions.</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design13.webp" alt="design13">
          <p class="figurecaption">Figure 13 schematic diagram of pDawn-CheZ</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design14.webp" alt="design14">
          <p class="figurecaption">Figure 14 schematic diagram of pDawn-CheZ-Flag</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design15.webp" alt="design15">
          <p class="figurecaption">Figure 15 schematic diagram of pDawn-CheZ-sfGFP</p>

          <h4>GROUP
            4:
            pSR43.6r&pSR58.6、pSR43.6r&pSR58.6-CheA、pSR43.6r&pSR58.6-CheA-Flag、pSR43.6r&pSR58.6-CheY、pSR43.6r&pSR58.6-CheY-Flag
          </h4>
          <pp class="sub">Group 4 consists of five groups of dual-plasmid systems. Under green light induction,
            pSR43.6r&pSR58.6 can express sfGFP, while the remaining four groups express CheA, CheA-Flag, CheY, and
            CheY-Flag respectively.</pp>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design16.webp" alt="design16">
          <p class="figurecaption">Figure 16 schematic diagram of pSR43.6r</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design17.webp" alt="design17">
          <p class="figurecaption">Figure 17 schematic diagram of pSR58.6-CheY</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design18.webp" alt="design18">
          <p class="figurecaption">Figure 18 schematic diagram of pSR58.6-CheA</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design19.webp" alt="design19">
          <p class="figurecaption">Figure 19 schematic diagram of pSR58.6-CheY-Flag</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design20.webp" alt="design20">
          <p class="figurecaption">Figure 20 schematic diagram of pSR58.6-CheA-Flag</p>

          <h4>GROUP 5: pDawn-CheZ-sfGFP、pSR43.6r&pSR58.6-CheA-sfGFP、pSR43.6r&pSR58.6-CheY-sfGFP</h4>
          <pp class="sub">In group 5, based on groups 3 and 4, plasmid systems for sfGFP-fused CheZ protein, sfGFP-fused
            CheA protein, and sfGFP-fused CheY protein have been constructed. This is aimed at more clearly localizing
            bacterial positions and observing flagellar movement. The system is currently under improvement.</pp>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design21.webp" alt="design21">
          <p class="figurecaption">Figure 21 schematic diagram of pDawn-CheZ-sfGFP</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design22.webp" alt="design22">
          <p class="figurecaption">Figure 22 schematic diagram of pSR58.6-CheA-sfGFP</p>

          <img src="https://static.igem.wiki/teams/5972/uploads/page/design/design23.webp" alt="design23">
          <p class="figurecaption">Figure 23 schematic diagram of pSR58.6-CheY-sfGFP</p>


        </section>

        <!-- ID：references -->
        <section id="references">
          <h1>References</h1>
          <p class="sub">[1] 王启要. 中国鱼类疫苗技术研发及应用研究进展[J]. 大连海洋大学学报,2022,37(1):1-9.</p>
          <p class="sub">[2] Kojima S, Blair DF. The bacterial flagellar motor: structure and function of a complex
            molecular machine. Int Rev Cytol. 2004;233:93-134.</p>
          <p class="sub">[3] Tan J, Zhang X, Wang X, et al. Structural basis of assembly and torque transmission of the
            bacterial flagellar motor. Cell. 2021 May 13;184(10):2665-2679.e19.</p>
          <p class="sub">[4] Zhang J, Luo Y, Poh CL. Blue Light-Directed Cell Migration, Aggregation, and Patterning. J
            Mol Biol. 2020 May 1;432(10):3137-3148. </p>
          <p class="sub">[5] Tan J, Zhang L, Zhou X, et al. Structural basis of the bacterial flagellar motor rotational
            switching. Cell Res. 2024 Nov;34(11):788-801.</p>
          <p class="sub">[6] Perlova T, Gruebele M, Chemla YR. Blue Light Is a Universal Signal for Escherichia coli
            Chemoreceptors. J Bacteriol. 2019 May 8;201(11):e00762-18.</p>
          <p class="sub">[7] Schmidl S R, Sheth R U, Wu A, et al. Refactoring and optimization of light-switchable
            Escherichia coli two-component systems[J]. ACS synthetic biology, 2014, 3(11): 820-831.</p>
          <p class="sub">[8] Kurzthaler C, Zhao Y, Zhou N, et al. Characterization and control of the run-and-tumble
            dynamics of Escherichia coli[J]. Physical Review Letters, 2024, 132(3): 038302.</p>
          <p class="sub">[9] Soffer G, Perry J M, Shih S C C. Real-time optogenetics system for controlling gene
            expression using a model-based design[J]. Analytical Chemistry, 2021, 93(6): 3181-3188.</p>
          <p class="sub">[10] Clavier T. Contrôle génétique de la croissance et des conditions de culture pour maximiser
            la production biotechnologique chez E. coli[D]. Université Grenoble Alpes, 2024.</p>
          <p class="sub">[11] Kumar S, Khammash M. Platforms for optogenetic stimulation and feedback control[J].
            Frontiers in bioengineering and biotechnology, 2022, 10: 918917.</p>
          <p class="sub">[12] Zhang Y, Xue X, Fang M, et al. Upconversion optogenetic engineered bacteria system for
            time-resolved imaging diagnosis and light-controlled cancer therapy[J]. ACS Applied Materials & Interfaces,
            2022, 14(41): 46351-46361.</p>
          <p class="sub">[13] Zhao R, Collins E J, Bourret R B, et al. Structure and catalytic mechanism of the E. coli
            chemotaxis phosphatase CheZ[J]. Nature structural biology, 2002, 9(8): 570-575.</p>
          <p class="sub">[14] Kennedy E N, Barr S A, Liu X, et al. Azorhizobium caulinodans chemotaxis is controlled by
            an unusual phosphorelay network[J]. Journal of bacteriology, 2022, 204(2): e00527-21.</p>
          <p class="sub">[15] Ou B, Lv H, Ge H, et al. Deletion of the cheZ gene results in the loss of swimming ability
            and the decrease of adhesion ability to Caco-2 cells in Escherichia coli Nissle 1917[J]. Folia
            Microbiologica, 2023, 68(3): 395-402.</p>
          <p class="sub">[16] Karmakar R. State of the art of bacterial chemotaxis[J]. Journal of Basic Microbiology,
            2021, 61(5): 366-379.</p>
          <p class="sub">[17] Zhang X, Feng Z, Li H, et al. Ingestible optoelectronic capsules enable bidirectional
            communication with engineered microbes for controllable therapeutic interventions[J]. Nature Microbiology,
            2025: 1-13.</p>
          <p class="sub">[18] Wadhwa N, Berg H C. Bacterial motility: machinery and mechanisms[J]. Nature reviews
            microbiology, 2022, 20(3): 161-173.</p>
          <p class="sub">[19] Zhulin I B, Berry M A, Andrianova E P. Diverse domain architectures of CheA histidine
            kinase, a central component of bacterial and archaeal chemosensory systems[J]. 2023.</p>
          <p class="sub">[20] Li X, Zhang C, Xu X, et al. A single-component light sensor system allows highly tunable
            and direct activation of gene expression in bacterial cells. Nucleic Acids Res. 2020 Apr 6;48(6):e33.</p>
        </section>

        <div class="logo-ring"></div>

      </div>
    </main>
    {% endblock %}